.codecov.yml

```diff
@@ -1,24 +1,30 @@
-# Documentation: https://github.com/codecov/support/wiki/Codecov-Yaml
+# Documentation: https://docs.codecov.io/docs/codecov-yaml
 
 codecov:
-  # CodeCov should only wait for CI's that run coverage tests
-  # DAutoTester and auto-tester are not in the default list
-  ci:
-    - "circleci.com"
   notify:
     after_n_builds: 1  # send notifications after the first upload
   bot: dlang-bot
 
 coverage:
   precision: 3
   round: down
-  range: 80...100
+  range: "80...100"
 
+  # Learn more at https://docs.codecov.io/docs/commit-status
   status:
-    # Learn more at https://docs.codecov.io/docs/commit-status
-    project: true
-    patch: true
-    changes: false
+    project:
+      default:
+        informational: true
+        # Informational doesn't work with project yet
+        threshold: 0.1
```

Comment:
I think 0.1% is a reasonable threshold to start with. If it's too low, we can always increase it?